packages/airnode-examples/package.json

```diff
@@ -35,23 +35,23 @@
     "@api3/airnode-deployer": "^0.4.0",
     "@api3/airnode-node": "^0.4.0",
     "@api3/airnode-protocol": "^0.4.0",
-    "ethers": "^5.4.4",
+    "ethers": "^5.4.5",
     "is-wsl": "^2.2.0"
   },
   "devDependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@nomiclabs/hardhat-waffle": "^2.0.1",
     "@types/jest": "^27.0.2",
-    "@types/node": "^16.10.3",
+    "@types/node": "^15.12.2",
     "@types/prompts": "^2.0.14",
     "chalk": "^4.1.2",
     "dotenv": "^10.0.0",
     "ethereum-waffle": "^3.4.0",
-    "hardhat": "^2.6.0",
-    "jest": "^27.2.5",
+    "hardhat": "^2.6.5",
+    "jest": "^26.6.3",
```

Comment:
Quite a few packages seem to have been downgraded (sometimes a major version like this). Any reason for that?

Reply:
I've added a check to use the same versions of shared dependencies in the monorepo (see: https://github.com/JamieMason/syncpack#fix-mismatches). The advantage is that:
a) less space reserved in node modules (no one cares)
b) "Predictability" - e.g. I know that jest in package X behaves the same as in Y (this is pretty good for example for typescript).

And I did the "easy" way and used the version that was used in majority of the packages.

Reply:
Oh right - I missed the syncpack part. That's been a concern of mine for a while and it's great to have a new check in place 💯